src/module/actor/monster-sheet.js

```diff
@@ -303,6 +304,13 @@ export class OseActorSheetMonster extends OseActorSheet {
       actorObject.rollAppearing({ event: ev, check: check });
     });
 
+    html.find(".treasure-table a").contextmenu((ev) => {
```

Comment:
Cool! A novel solution for removing the treasure table here. At some point we'll revisit if we can add a "are you sure" because I imagine an accidental right click would be really annoying